### ISSUE TITLE
Multi-level select field has no label

### ISSUE THREAD

**Issue by mathetos** [WordImpress/Give#331]
## A11Y Report Notes
http://b8d.3e8.myftpupload.com/donations/multi-level-w-select-field-opens-in-modal/
**Multi-Level w/ Select Field & Opens in Modal**
* Error /  priority 89%
* Form element has no label
* (line: 133, 3)

```<select id="give-donation-level-26" class="give-select give-select-level"><option data-price-id="1" class="give-donation-level-26" value="5.00">Small</option><option data-price-id="2" class="give-donation-level-26" value="10.00">M```

Provide a label for all controls. This allows all users to understand the type of information expected in the field.

## Possible Solutions
* Can we add a hidden label?
* Are there other methods for labelling the input without having the actual text spit out above the field that also suits the WCAG 2.0 Requirements?

Need Jordan's input on this one.

**Comment by DevinWalker** [comment]
I was thinking a hidden label too, but would it still be accessible?

**Comment by mathetos** [comment]
Jordan said he'd be able to chime in next week on our questions both in Slack and here.

**Comment by mathetos** [comment]
@TheGeniusWM Again, can we add a hidden label to address this accessibility issue?

**Comment by TheGeniusWM** [comment]
Yes, a hidden label will fix this.
